### PR TITLE
Neon number added

### PR DESCRIPTION
**Pull request by ankan1811**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

[#2463 ](https://github.com/TesseractCoding/NeoAlgo/issues/2463)

### CODE REVIEW

**Comment by ankan1811** [comment]
@HarshCasper @siddharth25pandey  Please review my PR. (The issue  [#2463](https://github.com/TesseractCoding/NeoAlgo/issues/2463) was created by me for C++ 4 days ago. But @ananddasani  had used the term (fixes <issue -number>) in his PR for java. So the issue got closed automatically.)

File: C-Plus-Plus/math/neon_number.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+//Function that prints the square of a number
+int square(int z)
+{
+    return z * z;
+}
```
Comment (siddharth25pandey) [inline C-Plus-Plus/math/neon_number.cpp]: There is no need to create additional function, C++ supports power in-built function

File: C-Plus-Plus/math/neon_number.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+/*
+		Input: 1 10000
+		
+		Output: 1 9
+		
+*/
+
+/*
```
Comment (siddharth25pandey) [inline C-Plus-Plus/math/neon_number.cpp]: Remove the blank lines

File: C-Plus-Plus/math/neon_number.cpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        return false;
+}
+
+int main()
+{
+    cout << "Enter the range:";
+    int a, b;
+    cin >> a >> b;
+
+    // Printing Neon Numbers according to the range
```
Comment (siddharth25pandey) [inline C-Plus-Plus/math/neon_number.cpp:38]: Print the statement what are you printing, for better user interface.
ex: The Neon numbers in the between _ and _ are :

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@ankan1811 Do the suggested changes

File: C-Plus-Plus/math/neon_number.cpp (hunk 3 above)
Comment (HarshCasper) [reply C-Plus-Plus/math/neon_number.cpp:38]: Resolve this @ankan1811

File: C-Plus-Plus/math/neon_number.cpp (hunk 1 above)
Comment (HarshCasper) [reply C-Plus-Plus/math/neon_number.cpp]: Resolve this @ankan1811

File: C-Plus-Plus/math/neon_number.cpp (hunk 2 above)
Comment (HarshCasper) [reply C-Plus-Plus/math/neon_number.cpp]: Resolve this @ankan1811

**Comment by ankan1811** [comment]
@siddharth25pandey Done

File: C-Plus-Plus/math/neon_number.cpp [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    cout<<"Neon numbers in between "<<a<<" to "<<b<<" are : \n";
+    for (int i = a; i <= b; i++)
+        if (Neon(i) == true)
+            cout << i << " ";
+}
+/*
+		Input: 1 10000	
+		Output: 1 9		
+*/
```
Comment (siddharth25pandey) [inline C-Plus-Plus/math/neon_number.cpp]: Update this according to your code

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@ankan1811 Do a minor modification

File: C-Plus-Plus/math/neon_number.cpp (hunk 4 above)
Comment (ankan1811) [reply C-Plus-Plus/math/neon_number.cpp]: @siddharth25pandey updated.Thanks!!!

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
@all-contributors bot please add @ankan1811 for code

**Pull request merged**